praw/models/reddit/emoji.py

```diff
@@ -0,0 +1,74 @@
+"""Provide the EMOJI class."""
+from ...const import API_PATH
+from .base import RedditBase
+
+
+class Emoji(RedditBase):
+    """An individual Emoji object."""
+
+    __hash__ = RedditBase.__hash__
+
```

Comment:
Can you add a line that says `STR_FIELD = 'name'`? This class variable is used in classes like [`Subreddit`](https://github.com/praw-dev/praw/blob/5ee4b1820c2591117e32be45778372e7c03a5f56/praw/models/reddit/subreddit.py#L59) that inherit from `RedditBase` for the [`__str__`](https://github.com/praw-dev/praw/blob/5ee4b1820c2591117e32be45778372e7c03a5f56/praw/models/reddit/base.py#L62-L64) and [`__repr__`](https://github.com/praw-dev/praw/blob/5ee4b1820c2591117e32be45778372e7c03a5f56/praw/models/reddit/base.py#L57-L60) methods. It's neccessary in `Emoji` too.